lab_1/main.py

```diff
@@ -69,7 +107,28 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    # here goes input parameters checks
+    if not isinstance(tokens, list) or not isinstance(word, str)\
+            or None in tokens:
+        return []
+    if left_context_size is True or right_context_size is True\
```

Comment:
No need for explicit check.
You can just `if var: ...`

Reply:
Here explicit check for bool value `True` is required, because this check is not for just having value in `left_context_size` and `right_context_size` variables, but for explicit bool value `True` check.  Test `test_get_concordance_bad_inputs` uses `True` as bad input. So, I had to check it explicitly.

Reply:
Alright, i get it now. Resolved.